test/distributed/test_overlap_bucketing_unit.py

```diff
@@ -719,6 +720,196 @@ def func(a, b, c):
             "%all_gather_into_tensor_out"
         ).run(graph_str)
 
+    def test_split_mm(self):
```

Comment:
This file doesnt actually test numerics, so it would be good to add to test_aten_comm_compute_reordering.py as well or test_inductor_distributed which actually runs multi-node. but this file is nice for modularity and quicker to run.